collector/zfs.go

```diff
@@ -35,9 +35,25 @@ type zfsSubsystemName string
 
 const (
 	arc            = zfsSubsystemName("zfsArc")
+	dmuTx          = zfsSubsystemName("zfsDmuTx")
+	fm             = zfsSubsystemName("zfsFm")
+	vdevCache      = zfsSubsystemName("zfsVdevCache")
+	xuio           = zfsSubsystemName("zfsXuio")
+	zfetch         = zfsSubsystemName("zfsFetch")
+	zil            = zfsSubsystemName("zfsZil")
 	zpoolSubsystem = zfsSubsystemName("zfsPool")
 )
 
+var subsystems = []zfsSubsystemName{
```

Comment:
Ah, here I was just thinking that go developers liked global variables...

Inlining complete!